### ISSUE TITLE
zynq 7015 | MYD-C7Z015 board

### ISSUE THREAD

**Issue by vladoshub** [SymbiFlow/prjxray#1626]
Hello everyone!
can you add support for this facility to the project?

**Comment by litghost** [comment]
We have documentation on how to do this: https://github.com/SymbiFlow/prjxray/blob/master/docs/db_dev_process/newpart.rst

Feel free to take a shot!

**Comment by vladoshub** [comment]
> We have documentation on how to do this: https://github.com/SymbiFlow/prjxray/blob/master/docs/db_dev_process/newpart.rst
> 
> Feel free to take a shot!
> 
> > <img alt="" width="48" src="https://repository-images.githubusercontent.com/114421621/68e8c380-e157-11e9-83b1-7c985f136526"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[SymbiFlow/prjxray](https://github.com/SymbiFlow/prjxray)**Documenting the Xilinx 7-series bit-stream format. - SymbiFlow/prjxray

yes, I saw it. but I really don’t understand where to get the data for "XRAY_IOI3_TILES" for the settings file. Thanks in advance
